ChatEventHandler.py

```diff
@@ -112,15 +111,7 @@ def typemethod(data):
 
 def format_msg(data):
     '''formats the msg text string properly'''
-    msg = ''
-
-    # gets the message list amount
-    list_count = len(data['message']['message'])
-
-    # loops though each message list entries
-    for counter in range(0, list_count):
-        # joins the message together
-        msg += str(data['message']['message'][counter]['text'])
+    msg = ''.join(item["text"] for item in data["message"]["message"])
```

Comment:
*Almost* perfect. Add a space between the quotes. `.join` joins all the items in a list by the delimiter, which in this case, is the text between the `''` :D